arachni/redis.conf

```diff
@@ -0,0 +1,1052 @@
+# Redis configuration file example.
```

Comment:
This file is very large (1050 lines). It's hard to tell what are the differences from upstream/distro config. It's making it very difficult to review/audit.

Is it absolutely necessary to include this here? Can you use a stock config instead? Otherwise, can you make a minimal version that is easier to review?